cognite_toolkit/cdf_tk/load.py

```diff
@@ -227,168 +228,13 @@ def delete(self, ids: Sequence[T_ID]) -> int:
     def retrieve(self, ids: Sequence[T_ID]) -> T_ResourceList:
         return self.api_class.retrieve(ids)
 
-    def load_file(self, filepath: Path, ToolGlobals: CDFToolConfig) -> T_Resource | T_ResourceList:
+    def load_resource(self, filepath: Path, ToolGlobals: CDFToolConfig) -> T_Resource | T_ResourceList:
         raw_yaml = load_yaml_inject_variables(filepath, ToolGlobals.environment_variables())
         if isinstance(raw_yaml, list):
             return self.list_cls.load(raw_yaml)
         return self.resource_cls.load(raw_yaml)
 
 
-@final
-class TimeSeriesLoader(Loader[str, TimeSeries, TimeSeriesList]):
```

Comment:
Reordering is necessary as `TimeSeriesLoader` is dependent on `DataSetLoader`, so `DataSetLoader` must come before `TimeSeriesLoader` in the file.